install/charts/dir/apiserver/templates/secret.yaml

```diff
@@ -1,7 +1,7 @@
 # Copyright AGNTCY Contributors (https://github.com/agntcy)
 # SPDX-License-Identifier: Apache-2.0
 
-{{- if .Values.privKey -}}
+{{- if or .Values.privKey (and .Values.syncAuthSecret (and .Values.syncAuthSecret.username .Values.syncAuthSecret.password)) (and .Values.ociAuthSecret (and .Values.ociAuthSecret.username .Values.ociAuthSecret.password)) -}}
```

Comment:
can you please add and document these details in the values file? i would suggest moving to a dedicated struct:

```
secrets:
	<secret-name>: <secret-value>
```

and check with 


```suggestion
{{- if .Values.secrets -}}
```

Reply:
I've implemented the suggested `secrets` refactoring, but had to revert since CodeQL flags any empty password strings in configuration files as security vulnerabilities 😢